### ISSUE TITLE
pg viewer doesn't plot data corretly

### ISSUE THREAD

**Issue by adamchengtkc** [gimli-org/gimli#421]
### Problem description

I am playing with the Heat Equation in 2D tutorial. https://www.pygimli.org/_tutorials_auto/2_modelling/plot_5-mod-fem-heat-2d.html#sphx-glr-tutorials-auto-2-modelling-plot-5-mod-fem-heat-2d-py

I tried changing the temperature for the bc `T = pg.solver.solveFiniteElements(mesh, a={1: 1.0, 2: 2.0, 3: 3.0, 4:3}, bc={'Dirichlet': {8: 100.0, 4: 50}}, verbose=True)` and inspected the result that T has a max of 100 and min of 50. But when I plot the data, the temperature field is still between 0 and 1. Am I missing anything? Thanks

### Your environment

**Operating system**: Debain bullseye
**Python version**: 3.9.5
**pyGIMLi version**: 1.2.6+96.gfd32b210
**Way of installation**: compiled from source from the latest git (last commit: fd32b210)

### Steps to reproduce

Everything is the same as the tutorial except
```python
T = pg.solver.solveFiniteElements(mesh,
                                  a={1: 1.0, 2: 2.0, 3: 3.0, 4:3},
                                  bc={'Dirichlet': {8: 100.0, 4: 50}}, verbose=True)
...
```

### Expected behavior

The colour field should how the correct temperature

### Actual behavior

Resulted figure
![Screenshot 2022-08-12 10 20 52](https://user-images.githubusercontent.com/52572642/184314645-00d6f817-ad5d-464a-93e7-0a7a719ed3d2.png)

> If possible, please add one or more labels to your issue, e.g. if you expect that your issue is rather a question than a problem with the code, please add the label "question".

**Comment by halbmy** [comment]
For me it works as supposed:
![image](https://user-images.githubusercontent.com/7136401/184331608-315142e9-b430-44cd-a67e-2e4a2162c8b3.png)
I am having the recent git but with pgcore 1.2.6.

**Comment by carsten-forty2** [comment]
same like halbmy .. works for me as expected with my local  my version (1.2.6+111.g02564fc5 (dev))

```
T = pg.solver.solveFiniteElements(mesh,
                                  a={1: 1.0, 2: 2.0, 3: 3.0, 4:3},
                                  bc={'Dirichlet': {8: 100.0, 4: 50}}, verbose=True)
ax, _ = pg.show(mesh, data=T, label='Temperature $T$', cMap="seismic", nCols=8)

pg.show(geom, ax=ax, fillRegion=False)
```
![image](https://user-images.githubusercontent.com/6043771/184336456-d77090d3-bcda-439f-ac1f-a8a1b2721047.png)

Interestingly the field distribution looks correct and just the cBar values seems wrong, maybe a weird caching problem? Did you tried to reset the script?

Whats the median of the calculated Temperatures?

```
import numpy as np
print(np.median(T))
# mine 69.05178497337528
```

**Comment by adamchengtkc** [comment]
I have tested passing random data to the data kwarg. And it show up with value scaled between 1 and 0. I am building the v1.2.6 and will report back

**Comment by adamchengtkc** [comment]
I can confirm it is working on my build for v1.2.6. Will try to do a diff between v1.2.6 and 1.2.6+96.gfd32b210 when I have time and report back

**Comment by halbmy** [comment]
We just released v1.3.0 so it would be good to see whether the problem is solved.

**Comment by adamchengtkc** [comment]
v1.3.0 compiled from source. works. No problem
